src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -151,12 +171,20 @@ let inline private aidFlattening
       [ anonContext.BucketSeed; aidContributions |> Seq.map fst |> seedFromAidSet ]
       |> generateNoise anonParams.Salt "noise" noiseSD
 
+    let outliers =
```

Comment:
Two comments:
1. How about we call this `outlierFlatteningByAid` or something more verbose across the board?
2. Can we move this bit up and substitute `outlierReplacement` and `flattening` for summing of this array? I have a feeling like we're calculating the same thing twice here